monitoring/api/AlertSample/Program.cs

```diff
@@ -359,5 +396,29 @@ static object EnablePolicies(string projectId, string filter, bool enable)
         }
         // [END monitoring_alert_enable_policies]
         // [END monitoring_alert_disable_policies]
+
+        static void DeleteAlertPolicy(string policyNameString)

```

Comment:
region tag for this and the next one?

Reply:
Added a region tag for deleting the notification channel.  No such tag was requested (by tech writers or others) for deleting a policy.

Reply:
I just needed the code to clean up a test project.